src/views/map.c

```diff
@@ -2879,7 +2883,7 @@ static void _dbscan_spread(unsigned int index)
   for(unsigned int i = 0; i < db.spreads->num_members; i++)
   {
     dt_geo_position_t *d = &db.points[db.spreads->index[i]];
-    if(d->cluster_id == NOISE || d->cluster_id == UNCLASSIFIED)
+//    if(d->cluster_id < 0) // NOISE or UNCLASSIFIED - but only those are included in the list anyway
```

Comment:
Probably to be removed.

Reply:
@ralfbrown : Can you comment on this? Should it be removed or is it a left over of some debug session?

I can remove it myself if needed, but I prefer to ask. TIA.

Reply:
Sorry, missed the comment previously.  I can do it over the weekend unless you're in a hurry....

Reply:
Nothing urgent, I'll wait for the weekend. Thanks!

Reply:
Done.  Also rebased.